examples/python/api_demo/main.py

```diff
@@ -277,18 +279,51 @@ def run_extension_component() -> None:
     )
 
 
+def run_image_tensors() -> None:
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    img_path = f"{dir_path}/rerun_logo_color_alpha.png"
+    img_bgra = cv2.imread(img_path, cv2.IMREAD_UNCHANGED)
+
+    img_rgba = cv2.cvtColor(img_bgra, cv2.COLOR_BGRA2RGBA)
+    rr.log_image("img_rgba", img_rgba)
+    img_rgb = cv2.cvtColor(img_rgba, cv2.COLOR_RGBA2RGB)
+    rr.log_image("img_rgb", img_rgb)
+    img_gray = cv2.cvtColor(img_rgb, cv2.COLOR_RGB2GRAY)
+    rr.log_image("img_gray", img_gray)
+
+    dtypes = [
+        "uint8",
+        "uint16",
+        "uint32",
+        "uint64",
+        "int8",
+        "int16",
+        "int32",
+        "int64",
+        "float16",
+        "float32",
+        "float64",
+    ]
+
+    for dtype in dtypes:
+        rr.log_image(f"img_rgba_{dtype}", img_rgba.astype(dtype))
+        rr.log_image(f"img_rgb_{dtype}", img_rgb.astype(dtype))
+        rr.log_image(f"img_gray_{dtype}", img_gray.astype(dtype))
+
+
 def main() -> None:
     demos = {
         "2d_lines": run_2d_lines,
         "3d_points": run_3d_points,
+        "bbox": run_bounding_box,
+        "extension_components": run_extension_component,
+        "image_tensors": run_image_tensors,
```

Comment:
can we please disable it by default? It drowns out everything else when one runs the demo without arguments (which I do a lot for quick swoop over all primitives)

Reply:
sure